fn/concurrency.go

```diff
@@ -71,8 +97,7 @@ func ParSliceErrCollect[V any](ctx context.Context, s []V,
 	// The goroutines that are executing in parallel should not return an
 	// error, but the Wait call may return an error if the context is
 	// canceled or timed out.
-	err := errGroup.Wait()
-	if err != nil {
+	if err := errGroup.Wait(); err != nil {
```

Comment:
Regarding the underlying function: does this behave as intended?

If the context is cancelled or times out, then it looks like every goroutine returns nil, so errGroup.Wait() will also return nil? I think the result is that we'll return a (possibly-partial) instanceErrors map and nil, but the intent here seems to be that we should return an error?